tests/testthat/test_base_generateFilterValuesData.R

```diff
@@ -150,3 +150,31 @@ test_that("ensemble methods work", {
   fi = generateFilterValuesData(multiclass.task, method = list("E-min", c("FSelectorRcpp_gain.ratio", "FSelectorRcpp_information.gain")))
   expect_true(all(!is.na(fi$data$value) == TRUE))
 })
+
+test_that("Ensemble filters select the top features", {
+library(survival)
+
+data(veteran)
+vet.task <- makeSurvTask(id = "VET", data = veteran, target = c("time", "status"))
+vet.task <- createDummyFeatures(vet.task)
+
+set.seed(24601)
+filt = mlr:::.FilterEnsembleRegister[["E-mean"]]
+cox.lrn <- makeLearner(cl="surv.coxph", id = "coxph", predict.type="response")
+dat = filt$fun(vet.task, 
+               base.methods = c("univariate.model.score", "randomForestSRC_importance"),
+               nselect = 5,
+               more.args = list("univariate.model.score"=list(perf.learner=cox.lrn))
+)
+ordered_dat = dat[with(dat, order(method, -value))]
+top_features1 = ordered_dat[ordered_dat$method == "E-mean"]$name[1:5]
```

Comment:
I am still not fully understanding what's happening here. Based on your comparison later, this will be the "true" reference. How can we be sure that the internal functions do a correct ranking only based on this code?

This code is also using 

https://github.com/mlr-org/mlr/blob/27f0ba48a11ca979e9a20dfec9b1df769e83f827/R/FilterEnsemble.R#L275-L277

at a certain stage, isn't it?

I think the only robust check we can do here is to check whether the code we are using for ranking/ordering the `rank` and `value` columns (see above) is working correctly by checking that these columns are in a asc/desc order afterwards?

Reply:
Yes I think you are right that this is not an adequate test. Do we need to check the output from calcBaseFilters, after ranking?